inference-engine/src/inference_engine/src/ie_core.cpp

```diff
@@ -1160,3 +1171,218 @@ void Core::UnregisterPlugin(const std::string& deviceName_) {
 }
 
 }  // namespace InferenceEngine
+
+namespace ov {
+namespace runtime {
+
+class Core::Impl: public InferenceEngine::Core::Impl {};
```

Comment:
can we do in opposite case? Because in future we need:
1. ov::Core::read_network to read network as is
2. ie::Core::ReadNetwork to read network first, then perform transformation to old behavior